file.js

```diff
@@ -1,68 +1,74 @@
 const db = new BaseDatos();
 // para prueba de no conectado
-if (new Date().getTime()%2){
-db.connect();}
+if (new Date().getTime() % 2) {
+  db.connect();
+}
 
-
-function leerDesdeDB   (){
-try{Utils.disableAll();
-const id = prompt('ID a leer:');
-const data = db.read(id);
-Utils.enableAll();
-return data;}
-catch(err){// error
-//alert('Error- Vuelva a  ingresar el ID a leer');
-alert(`Error- Vuelva a  ingresar el ID a leer!\n\n\t${err.message}`);
-console.warn('Error- Vuelva a  ingresar el ID a leer',err);
-//se vuelve a habilitar el boton
-// Utils.enableAll();
-return null;}finally{Utils.enableAll();}}
+function leerDesdeDB() {
+  try {
+    Utils.disableAll();
+    const id = prompt("ID a leer:");
+    const data = db.read(id);
+    Utils.enableAll();
+    return data;
+  } catch (err) {
+    //alert('Error- Vuelva a  ingresar el ID a leer');
+    alert(`Error- Vuelva a  ingresar el ID a leer!\n\n\t${err.message}`);
+    console.warn("Error- Vuelva a  ingresar el ID a leer", err);
+    //se vuelve a habilitar el boton
```

Comment:
Se deja un espacio luego de las `//`.
```suggestion
    // se vuelve a habilitar el boton
```